### ISSUE TITLE
Lower fine-tuning accuracy using MatConvNet

### ISSUE THREAD

**Issue by MichaelLiang12** [vlfeat/matconvnet#1044]
I use MatConvNet to fine-tune pre-trained models ( vgg-16, vgg-19 or res-50) on my own dataset (Cub bird 2011). But I get lower accuracies for all three models compared with those reported in previous papers ( using Caffe .etc). 

What's wrong with it? It is because I use bad super-parameters ( though I have tried different parameters) or MatCovNet's own problem?

Any one finetuned these models? Do you get a comparable accuracy ?
